<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>workhours</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>399</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>176</y>
      <width>110</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Day length total</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_length_total_hh">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>216</y>
      <width>37</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>216</y>
      <width>18</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>hh</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_length_total_mm">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>216</y>
      <width>37</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>216</y>
      <width>26</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_length_total_dec">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>216</y>
      <width>55</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>216</y>
      <width>53</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>h in dec</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>255</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Day length without lunch</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_length_no_lunch_hh">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>295</y>
      <width>37</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>295</y>
      <width>18</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>hh</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_length_no_lunch_mm">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>295</y>
      <width>37</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>295</y>
      <width>26</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_length_no_lunch_dec">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>295</y>
      <width>55</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>295</y>
      <width>53</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>h in dec</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>69</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Start time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_start_hh">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>39</y>
      <width>96</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_start_mm">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>39</y>
      <width>96</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>70</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>End time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_end_hh">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>110</y>
      <width>96</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>39</y>
      <width>18</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>hh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>110</y>
      <width>18</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>hh</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_end_mm">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>110</y>
      <width>96</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>39</y>
      <width>26</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>26</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="gif_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/fonts/piis_iso.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>417</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
